nemo/collections/tts/losses/fastspeech2loss.py

```diff
@@ -43,6 +43,48 @@ def forward(self, *, log_duration_pred, duration_target, mask):
         return torch.nn.functional.mse_loss(log_duration_pred, log_duration_target)
 
 
+class L1MelLoss(Loss):
+    """A Loss module that computes L1 mel loss for FastSpeech 2."""
+
+    @property
+    def input_types(self):
+        return {
+            "spec_pred": NeuralType(('B', 'C', 'T'), MelSpectrogramType()),
```

Comment:
I think the order of the dimensions of L1MelLoss should match the `spec_pred` input of L2MelLoss to prevent confusion. Can you either change this to (B, T, C) or change the L2MelLoss input to (B, C, T) as well (+ corresponding FS2 code)?

Reply:
Changed all to B, C, T